aws/eb_rds/main.tf

```diff
@@ -0,0 +1,86 @@
+# Create a security group for the beanstalk server
+resource "aws_security_group" "beanstalk" {
+  name   = "${var.eb_application}-${var.eb_environment}-server-sg"
+  vpc_id = "${module.vpc.vpc_id}"
+
+  ingress {
```

Comment:
I would restrict the inbound a bit more. Maybe only allow HTTP/S from anywhere and SSH only for a specific IP